include/pmacc/math/vector/Vector.hpp

```diff
@@ -525,9 +525,9 @@ struct Vector : private T_Storage<T_Type, T_dim>, protected T_Accessor, protecte
         return stream.str();
     }
 
-    HDINLINE dim3 toDim3() const
+    HDINLINE cupla::dim3 toDim3() const
```

Comment:
There is an issue with this change here and in other lines: the `dim3` macro is still available and is substituted first.

Reply:
solved by using `cupla.hpp` instead of the redefinition macros